synapse/lib/stormtypes.py

```diff
@@ -3959,6 +3959,11 @@ class Query(Prim):
          'type': {'type': 'function', '_funcname': '_methQueryExec',
                   'returns': {'type': ['none', 'any'],
                               'desc': 'A value specified with a return statement, or none.', }}},
+        {'name': 'size',
+         'desc': 'Execute the Query in a sub-runtime and return the number of nodes yielded.',
```

Comment:
Lets add a `max=` arg that can be used to control the loop and has a default value of like... 1000?

Reply:
Added, but named it `limit` instead, does that work? I also considered `maxn` but that seemed less intuitive.